Packs/ShiftManagement/ReleaseNotes/1_1_4.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+##### GetUsersOnCall
+- Added a new argument *include_OOO_users* which allows the user to choose if to include in the output also users
+ that are Out Of Office. By default, Out Of Office users are not included in the output.
+- Updated the docker image from `3.8.5.10845` to `3.9.1.14969`.
```

Comment:
The convention should be `Upgraded the Docker image to demisto/pyjwt3:1.0.0.8871.`